src/api/test/functional/group_test.rb

```diff
@@ -4,6 +4,11 @@ class GroupControllerTest < ActionDispatch::IntegrationTest
 
   fixtures :all
 
+  def setup
+    super
```

Comment:
On Freitag, 9. Oktober 2015, 03:48:07 CEST wrote Henne Vogelsang:
> > @@ -4,6 +4,11 @@ class GroupControllerTest < ActionDispatch::IntegrationTest
> >  
> >    fixtures :all
> >  
> > +  def setup
> > +    super
> 
> Why do you call super?

it should not ommit the generic setup method from derived classes, but extend them

-- 

Adrian Schroeter
email: adrian@suse.de

SUSE Linux GmbH, GF: Felix Imend�rffer, Jane Smithard, Graham Norton, HRB 21284 (AG N�rnberg) 
 
Maxfeldstra�e 5                         
90409 N�rnberg 
Germany